cips/cip-blob-verified-signer.md

```diff
@@ -57,7 +57,9 @@ Although no version changes are required for protobuf encoded blobs, share encod
 
 ![Diagram of V2 Share Format](../assets/cip-blob-verified-signer/blob-v2-share-format.svg)
 
-Blobs with an empty `signer` string would remain encoded using the v1 format. Note that in this diagram it is the `Info Byte` that contains the share version. Not to be confused with the namespace version.
+Note that in this diagram it is the `Info Byte` that contains the share version. Not to be confused with the namespace version.
+
+Blobs with an empty `signer` string would remain encoded using the v1 format. A transaction specifying a share version of 2 and an empty signer field would be rejected. Equally so, specifying a share version of 1 and a non empty signer field would be rejected.
```

Comment:
[nit] the current share version is `0` so a new share version would be `1`. To avoid confusion, we may want to replace v1 => 0 and v2 => 1 in these lines.

Ref: https://github.com/celestiaorg/celestia-app/blob/c5b0b9421e8bbe04d26104716bb86750a6891ab0/specs/src/specs/shares.md?plain=1#L49-L50

Reply:
let's do this and I think we can get this (1) assigned a CIP number (2) edit the image directory and path in .md (3) and merged

Reply:
Yes ok, I can also refer to them as v0 and v1

Reply:
Resolved in https://github.com/celestiaorg/CIPs/pull/147/commits/62e5ef1ac8ffb785e5777cd837439d24b8a201f6